modules/videoio/src/cap_msmf.cpp

```diff
@@ -710,7 +710,7 @@ _ComPtr<IMFAttributes> CvCapture_MSMF::getDefaultSourceConfig(UINT32 num)
     CV_Assert(num > 0);
     _ComPtr<IMFAttributes> res;
     if (FAILED(MFCreateAttributes(&res, num)) ||
-        FAILED(res->SetUINT32(MF_READWRITE_ENABLE_HARDWARE_TRANSFORMS, true)) ||
+        FAILED(res->SetUINT32(MF_READWRITE_ENABLE_HARDWARE_TRANSFORMS, false)) ||
```

Comment:
This should not be done unconditionally by default.

Not all Users have such buggy MSMF configurations. Only some Users are affected by this MSMF bug (should be handled by MS, BTW).

It is not a clear fix as it introduces new regressions. So we can accept here workaround only.

Introduce runtime option `OPENCV_VIDEOIO_MSMF_DISABLE_HW_TRANSFORMS=0` through `utils::getConfigurationParameterBool()` and use it here.

Reply:
Thanks for the feedback. I will do as you suggested.